LibTest/io/Stdin/first_A04_t01.dart

```diff
@@ -43,7 +43,10 @@ run_main() async {
     await process.stdout.toList().then((out) { res_elementAt = out[0]; });
   });
 
-  Expect.listEquals(res_elementAt, res_first);
+  Expect.isTrue("12345\n\n".startsWith(String.fromCharCodes(res_first).
```

Comment:
Please use `utf8.decode()` / `ascii.decode()` instead of `String.fromCharCodes`, ideally using the system encoding.